src/pages/product-design/brand-identity/index.md

```diff
@@ -3,13 +3,11 @@ title: Brand Identity
 ---
 ## Brand Identity
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/product-design/brand-identity/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Make sure you create a brand not only with the logo. Brand identity is about all usages and applications a brand may have passing its essence with ease.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+It's a manual showing where the identity may be inserted, based on design patterns.
```

Comment:
What is the "it" you say is a manual?

Reply:
I mean to have. Brand identity has a manual.